.github/workflows/try-in-web-ide.yaml

```diff
@@ -7,10 +7,10 @@ on:
 
 jobs:
   add-link:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     steps:
       - name: Web IDE Pull Request Check
         id: try-in-web-ide
-        uses: redhat-actions/try-in-web-ide@v1
+        uses: redhat-actions/try-in-web-ide@v1.2
```

Comment:
why 1.2?

Reply:
Because @v1 is still set to node 12: https://github.com/redhat-actions/try-in-web-ide/blob/v1/action.yml
and to get node 16 you have to specifically get 1.2 https://github.com/redhat-actions/try-in-web-ide/blob/v1.2/action.yml